MovieNight/ActorViewController.swift

```diff
@@ -64,4 +70,35 @@ class ActorViewController: UIViewController, UITableViewDataSource, UITableViewD
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         return actors.count
     }
+    
+    @IBAction func didSelectActor(_ sender: Any) {
+        let loveButton = sender as! UIButton
+        let indexPath = IndexPath(row: loveButton.tag, section: 0)
```

Comment:
This line is a bit risky. You don't need to change it. I just want you to be aware of the risk. You're assuming that there are no sections. If in the future the designer wants to add sections to the table view for whatever reason, this code would break in runtime.